build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -93,15 +107,22 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 		);
 
 		if (prExists) {
-			this.verbose(`Open pull request exists`);
-			return;
+			this.log(`Open pull request exists`);
+			this.exit(0);
 			// eslint-disable-next-line no-warning-comments
```

Comment:
`return` was added over here before because if the PR already exists then the GitHub workflow use to fail. But ideally it should succeed.

Reply:
Well, somewhere we have to check if the PR exists before we create it. If we don't check here, then the workflow will fail when we call createPullRequest, no?

Reply:
why would the workflow fail? 

It would result in the creation of duplicate main/next PR, where both PRs would appear identical.

Reply:
You're right - it wouldn't fail but there would be duplicates created. Still, I think exiting early is the right thing to do since the workflow now runs exclusively on a schedule.